distributions/otelcol-contrib/README.md

```diff
@@ -0,0 +1,16 @@
+# OpenTelemetry Collector Contrib Distro
+
+This distribution contains all the components from both the [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector) repository and the [OpenTelemetry Collector Contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib) repository. This distribution includes open source and vendor supported components.
+
+## Recommendation
+
+As this distribution contains many components, it is a good starting point to try various configurations. However, when running in production, it is recommended to limit the collector to contain only the components necessary for an environment. Some reasons to do this:
+
+* reduce the size of the collector, reducing deployment times for the collector
+* improve the security of the collector by reducing the available attack surface area
```

Comment:
Is this true?  I suppose components could have an `init()` function that did something malicious or opened an avenue for interaction, but in the normal course of things more components only means more factory functions in the service's factory map.  It's not until a component is used in a configuration that it is instantiated and could become a security concern.

Reply:
I think you're right @Aneurysm9, but I believe the sentiment here is "more components means more opportunity for a compromised component, which means more opportunity that you'll use a compromised component".  I think its all about increased chances that something could happen due to the large number of components that you could chose to use.

Reply:
@Aneurysm9 in theory if someone enabled remote configuration (as OpAmp progresses) I would expect this to be true, I'm happy to remove this if you disagree.

Reply:
Vulnerabilities are typically exploited by taking advantage of multiple factors. Having all possible components in there means that we are making an attacker's life easier. It's good practice to reduce the surface attack to only the things that are absolutely necessary.